run_games.py

```diff
@@ -79,13 +79,26 @@ def run_match(best, root_dir, c_chess_exe, concurrency, book_file_name, stockfis
         command = command + " -engine cmd={} name={} option.EvalFile={}".format(
             stockfish_exe, net, os.path.join(os.getcwd(), net)
         )
-    command = command + " -pgn {} 0 > {} 2>&1".format(
-        pgn_file_name, c_chess_out_file_name
+    command = command + " -pgn {} 0 2>&1".format(
+        pgn_file_name
     )
 
     print("Running match with c-chess-cli ... {}".format(pgn_file_name), flush=True)
-    ret = os.system(command)
-    if ret != 0:
+    c_chess_out = open(os.path.join(root_dir, "c_chess.out"), 'w')
+    process = subprocess.Popen("stdbuf -o0 " + command, stdout=subprocess.PIPE, shell=True)
```

Comment:
this might make it difficult to run the script on windows? @Sopel97 might want to comment

Reply:
maybe this can be achieved by passing `bufsize=0` to `Popen`? https://docs.python.org/3/library/subprocess.html#subprocess.Popen

Reply:
Unfortunately not, I tried that - I think it's because of shell=True, so it ends up setting the buffer size for the shell process.  I could try getting rid of shell=True so there isn't a wrapping shell process.

Reply:
actually, I realize c-chess-cli doesn't even support windows, so maybe this is a non-issue?